tests/unit/stylo/size_of.rs

```diff
@@ -57,5 +57,5 @@ size_of_test!(test_size_of_specified_image, specified::image::Image, 40);
 
 // FIXME(bz): These can shrink if we move the None_ value inside the
 // enum instead of paying an extra word for the Either discriminant.
```

Comment:
Is this fixed?

Reply:
I *think* so. That is, we’ll get the shrinkage benefit without changing the type definitions. I’ll double check tomorrow when Rust Nightly has this change, and remove the FIXME comment.